### ISSUE TITLE
Conflict with glossaries package

### ISSUE THREAD

**Issue by proafxin** [Jinwen-XU/colorist#9]
MWE:

```latex
\documentclass{book}
\usepackage{blindtext}
\usepackage{colorist}

%\usepackage{amsthm}
%\newtheorem{theorem}{Theorem}

\usepackage{hyperref}
\usepackage[xindy]{glossaries}

\makeglossaries

\begin{document}
	\blinddocument

	\appendix

	\chapter{Glossary}
	\begin{theorem}
		This is a theorem
	\end{theorem}

	\newglossaryentry{a}
	{
		name={An entry},
		description={An entry},
		symbol={\ensuremath{\mathbb{R}}}
	}
	\glsaddall
	\printglossaries

\end{document}
```

Errors:

```bash
66: Command \glossariescaptionsportuges already defined. }
: No file test.aux.
14: Font shape `T1/cmss/m/sc' in size <10> not available(Font) Font shape `T1/cmr/m/sc' tried instead
```

Output if `colorist` package is ignored:

```latex
\documentclass{book}
\usepackage{blindtext}
%\usepackage{colorist}

\usepackage{amsthm}
\newtheorem{theorem}{Theorem}

\usepackage{hyperref}
\usepackage[xindy]{glossaries}

\makeglossaries

\begin{document}
	\blinddocument

	\appendix

	\chapter{Glossary}
	\begin{theorem}
		This is a theorem
	\end{theorem}

	\newglossaryentry{a}
	{
		name={An entry},
		description={An entry},
		symbol={\ensuremath{\mathbb{R}}}
	}
	\glsaddall
	\printglossaries

\end{document}
```

[test.pdf](https://github.com/user-attachments/files/17541736/test.pdf)

**Comment by Jinwen-XU** [comment]
Sorry for the delay, I caught a cold last week and only noticed your message by weekend. The issue seems to be a bug of `glossaries`, as pointed by Javier Bezos (the maintainer of the package `babel`) in a comment to [this TeX.SX question](https://tex.stackexchange.com/questions/730002/command-glossariescaptionsportuges-already-defined-when-both-portuguese-and-bra) I posed. This is caused because I used the package `ProjLib` to setup multilingual configuration, in which both `portuguese` and `brazilian` are enabled which resulted in a conflict.

I shall consider putting a temporary patch to my package to resolve this issue. In the meantime, try what Javier suggested in the comment: adding the line
```tex
\AddToHook{file/glossaries-portuges.ldf/before}{\let\glossariescaptionsportuges\relax}
```
before loading `glossaries`.
